src/repositories/anchor-repository.ts

```diff
@@ -55,6 +55,7 @@ export class AnchorRepository implements IAnchorRepository {
   async findByRequestId(requestId: string): Promise<StoredAnchor | null> {
     const row = await this.table.where({ requestId: requestId }).first()
     if (!row) return null
+    console.log("IM HERE with ", row, requestId)
```

Comment:
Should we take this log out, @JulissaDantes? Might flood the logs at high throughput.

Reply:
You are completely right. Thank you for pointing this out!